src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -0,0 +1,116 @@
+import { Box, Button, EntityHeader, Flex, Sans } from "@artsy/palette"
+import { FeaturedArtists_collection } from "__generated__/FeaturedArtists_collection.graphql"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import { get } from "lib/utils/get"
+import React from "react"
+import { TouchableHighlight, TouchableWithoutFeedback } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface FeaturedArtistsProps {
+  collection: FeaturedArtists_collection
+}
+
+interface FeaturedArtistsState {
+  showMore: boolean
+}
+
+export class FeaturedArtists extends React.Component<FeaturedArtistsProps, FeaturedArtistsState> {
+  state = {
+    showMore: false,
+  }
+
+  handleTap = (context: any, href: string) => {
+    SwitchBoard.presentNavigationViewController(context, href)
+  }
+
+  getFeaturedArtistEntityCollection = (
+    artists: FeaturedArtists_collection["artworksConnection"]["merchandisableArtists"]
+  ) => {
+    return artists.map((artist, index) => {
+      const hasArtistMetaData = artist.nationality && artist.birthday
+      return (
+        <Box width="100%" key={index} pb={20}>
+          <TouchableWithoutFeedback onPress={() => this.handleTap(this, `/artist/${artist.slug}`)}>
+            <EntityHeader
+              imageUrl={artist.image.resized.url}
```

Comment:
#minor but you may want to use a safe `get` here in case one of these items returns null

Reply:
That feels like a smart move. I think we should make this change, unless someone can definitely say that artist always have images.

Reply:
Artists definitely sometimes have no images, we should use `get` here 👍

Reply:
For future reference, we can use optional chaining instead of `get` now!! 🎉 

```ts
artist.image?.resized.url
```

Reply:
I forgot all about optional chaining! Yay!